src/pages/groupComparison/MultipleCategoryBarPlot.tsx

```diff
@@ -74,6 +74,9 @@ const DEFAULT_BOTTOM_PADDING = 10;
 const LEGEND_ITEMS_PER_ROW = 4;
 const BOTTOM_LEGEND_PADDING = 15;
 const RIGHT_PADDING_FOR_LONG_LABELS = 50;
+export function getPathwayData() {
+    return this.data;
```

Comment:
This doesn't seem right. Reference to `this` outside a class method should not work as intended. As far as I can see we don't really call this function anywhere. We should remove it.